src/dodal/devices/slits.py

```diff
@@ -15,3 +15,17 @@ def __init__(self, prefix: str, name: str = "") -> None:
             self.x_centre = Motor(prefix + "X:CENTRE")
             self.y_centre = Motor(prefix + "Y:CENTRE")
         super().__init__(name)
+
+
+class SpecialSlitsi22(StandardReadable):
+    """
+    these are unlikely to be moved during a scan
+    """
+
+    def __init__(self, prefix: str, name: str = "") -> None:
+        with self.add_children_as_readables():
+            self.x = Motor(prefix + "X")
+            self.y = Motor(prefix + "Y")
+            self.horizontal_dso = Motor(prefix + "HDSO")
+            self.vertical_dso = Motor(prefix + "VDSO")
```

Comment:
Should: As someone not involved in i22, it's not clear what `dso` means here. Could we have a comment or rename the variable to something more human readable

Reply:
I do not either. that is how it was defined in EPICS. I did inquire on 07.08.2024 and received no response since.

Reply:
Ok, well I don't feel comfortable us having code in `dodal` where no-one understands what it means. Could you please chase up? Have you asked Scientists+Controls?

Reply:
I followed up again

Reply:
fixed!

 @DominicOram

Reply:
So they stand for `defining_slit_opening`? Can we just use `gap` as that's what we use everywhere else

Reply:
I'm also assuming that "defining slit opening" -> "slit width" -> "gap"?

Reply:
Yes, I think they are all equivalent, we use `gap` everywhere else, it's succinct (unlike `defining_slit_opening`) and makes more sense for x and y (unlike `width` which sort of implies just `x`). So I think we should stick to gap at the ophyd level

Reply:
Ok I change it into `x_gap` etc, and moving the 'dso' into the class docstring